src/Aspire.Dashboard/Components/Controls/LogViewer.razor.cs

```diff
@@ -32,7 +31,7 @@ public sealed partial class LogViewer
     public LogEntries? LogEntries { get; set; } = null!;
 
     [Parameter]
-    public bool ShowTimestamp { get; set; }
+    public TimestampDisplayMode TimestampDisplayMode { get; set; }
```

Comment:
I think these should be different bool parameters:

ShowTimestamp
IsTimestampUtc